src/Builder/FrontendDialectHelper.cpp

```diff
@@ -256,6 +256,31 @@ mlir::DenseElementsAttr onnxTensorProtoToDenseElmAttr(
         tensorType, llvm::makeArrayRef(arrayAttrInitializer));
     break;
   }
+  case (onnx::TensorProto::BOOL): {
+    // extend 1 bit data to 1 byte bool type
```

Comment:
Does this play nicely with other ONNX mlir code?
We might want to do this in our codebase, since it is unique to our hardware.

Reply:
I think this would agree with the ONNX mlir spec: The element type of DenseElementsAttr must be of integer type of bitwidth 1 according to DenseElementsAttr::getBoolValue (https://mlir.llvm.org/doxygen/classmlir_1_1DenseElementsAttr.html#acd41d2a6ba96e003105067ce7d9cea6a). 
The purpose of these lines is to convert TensorProto into DenseElementsAttr, and the walkaround here is to convert the 1-bit representation of bool value into actual bool so that I can use "ArrayRef< T > llvm::makeArrayRef".

Reply:
Ah I see. Thanks!